src/test/java/database/service/JsonDatabaseServiceTest.java

```diff
@@ -28,12 +29,14 @@
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-public class JsonDatabaseServiceTest {
+class JsonDatabaseServiceTest {
     private Student firstStudent;
     private Student secondStudent;
     private Student thirdStudent;
     private Student fourthStudent;
     private JsonDatabaseService jsonDatabaseService;
+    private final String INVALID_PARAMETER_VALUE = "Invalid parameter value. Limit value should be in(0..100), " +
```

Comment:
Please use `INVALID_PARAMETER_VALUE` constant from `JsonDatabaseService` and replace `MAX_LIMIT_VALUE` with your value.